backend/src/main/java/com/benine/backend/http/RequestHandler.java

```diff
@@ -45,6 +45,22 @@ public Attributes parseURI(String uri) throws MalformedURIException {
   }
 
   /**
+   * Formats the response message for a success or failure.
+   * @param exchange the HttpExchange.
+   * @param correct boolean that is true if the exchange is successful. False otherwise.
+   */
+  public void responseMessage(HttpExchange exchange, boolean correct) {
```

Comment:
I don't really agree with this. This method is created to get rid of all the String response = ..... in the code everywhere. Now you can just call this method and say if it is successful (true) or failed (false).